src/app/shared/users/user.interface.ts

```diff
@@ -5,3 +5,13 @@ export interface User {
     email?: string;
     password?: string;
 }
+export interface RemoteUser {
```

Comment:
User should remain the same interface. Imagine that you are in monorepo with shared models across BE and FE apps...

Reply:
it is redundant . deleted